new/LIT7237QeneFequrana.xml

```diff
@@ -0,0 +1,77 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7237QeneFequrana" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ፍቁራነ፡ ኮኑ፡ ወኢተሴሰዩ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Fǝqurāna konu wa-ʾitasessayu (Qǝne of the mawaddǝs-kʷǝllǝkǝmu-type)</title>
```

Comment:
Yes, it seems to be meweddes. Meweddes is a complex type and Guidi presents it also in a complex way.  I also understand that simpe meweddes is eight verses and meweddes kwellekemu is nine verses.